KingfisherWebP/Classes/Image+WebP.swift

```diff
@@ -69,20 +69,16 @@ extension KingfisherWrapper where Base: KFCrossPlatformImage {
 // MARK: - WebP Format Testing
 extension Data {
     public var isWebPFormat: Bool {
-        if count < 12 {
```

Comment:
I suppose it would be much neater that just change `if count < 12` to `if count < 12 || first != 0x52`, and leave the following lines unmodified.